aws/templates/solution/solution_ElasticSearch.ftl

```diff
@@ -349,5 +349,256 @@
             /]
         [/#if]
 
+        [#list occurrence.Occurrences![] as subOccurrence]
+
+            [#assign subCore = subOccurrence.Core ]
+            [#assign subSolution = subOccurrence.Configuration.Solution ]
+            [#assign subResources = subOccurrence.State.Resources ]
+
+            [#switch subCore.Type ]
+                [#case ES_DATAFEED_COMPONENT_TYPE ]
+
+                    [#assign streamId = subResources["stream"].Id ]
+                    [#assign streamName = subResources["stream"].Name ]
+                    
+                    [#assign streamRoleId = subResources["role"].Id ]
+                    [#assign streamRolePolicyId = formatDependentPolicyId(streamRoleId, "local")]
+
+                    [#assign streamSubscriptionRoleId = subResources["subscriptionRole"].Id!"" ]
+                    [#assign streamSubscriptionPolicyId = formatDependentPolicyId(streamSubscriptionRoleId, "local")]
+                    
+                    [#assign streamLgId = (subResources["lg"].Id)!"" ]
+                    [#assign streamLgName = (subResources["lg"].Name)!"" ]
+                    [#assign streamLgStreamId = (subResources["streamlgstream"].Id)!""]
+                    [#assign streamLgStreamName = (subResources["streamlgstream"].Name)!""]
+                    [#assign streamLgBackupId = (subResources["backuplgstream"].Id)!""]
+                    [#assign streamLgBackupName = (subResources["backuplgstream"].Name)!""]
+
+                    [#assign logging = subSolution.Logging ]
+                    [#assign encrypted = subSolution.Encrypted]
+
+                    [#assign streamProcessors = []]
+
+                    [#assign dataBucketPrefix = getAppDataFilePrefix(subOccurrence) ]
+
+                    [#if logging ]
+                        [#if deploymentSubsetRequired("lg", true) && isPartOfCurrentDeploymentUnit(streamLgId) ]
+                            [@createLogGroup
+                                mode=listMode
+                                id=streamLgId
+                                name=streamLgName /]
+                            
+                            [@createLogStream
+                                mode=listMode
+                                id=streamLgStreamId
+                                name=streamLgStreamName
+                                logGroup=streamLgName
+                                dependencies=streamLgId
+                            /]
+
+                            [@createLogStream
+                                mode=listMode
+                                id=streamLgBackupId
+                                name=streamLgBackupName
+                                logGroup=streamLgName
+                                dependencies=streamLgId
+                            /]
+
+                        [/#if]
+                    [/#if]
+
+                    [#list subSolution.LogWatchers as logWatcherName,logwatcher ]
```

Comment:
needs !{} ?

Reply:
The solution config should always have LogWatchers and it would be empty by default wouldn't it?